.github/workflows/update-stage0.yml

```diff
@@ -23,16 +24,17 @@ jobs:
     - uses: actions/checkout@v3
       with:
         ssh-key: ${{secrets.STAGE0_SSH_KEY}}
-    - name: Check if update is needed
+    - run: echo "DOIT=yes" >> "$GITHUB_ENV"
```

Comment:
I'm not super excited that the flag is named `DOIT` here!

Reply:
Happy to change, any suggestions?

Reply:
Is `should_update_stage0` better?

Reply:
Just unclear. (I have a vendetta against 'it'.)